### PR TITLE
Jobs page

### PR DESCRIPTION
**Pull request by BMillman19**
## Description

[Jobs page in staging
](http://staging-0xproject.s3-website-us-east-1.amazonaws.com/careers)
## Checklist:

*   [ ] Change requires a change to the documentation.
*   [ ] Added tests to cover my changes.
*   [ ] Added new entries to the relevant CHANGELOG.jsons.
*   [ ] Labeled this PR with the 'WIP' label if it is a work in progress.
*   [ ] Labeled this PR with the labels corresponding to the changed package.

### CODE REVIEW

File: packages/website/ts/pages/jobs/benefits.tsx [hunk 1]
```diff
@@ -1,109 +1,134 @@
...
+            ) : (
+                <Container className="flex-column">
+                    <BenefitsList />
+                    <Container marginTop="50px">
+                        <ValuesList />
+                    </Container>
+                </Container>
+            )}
+        </Container>
+    );
```
Comment (fragosti) [inline packages/website/ts/pages/jobs/benefits.tsx]: using the media queries abstraction tool seems better suited here.

What I've also seen done is expand the `Container` API to also take in an object for each (relevant) prop. For example:
```
<Container maxWidth={{
  small: 800,
}}>
... 
</Container>
```
Again, just a thought. Careers page is pretty isolated, so whatever works is good.

File: packages/website/ts/pages/jobs/mission.tsx [hunk 2]
```diff
@@ -8,49 +10,36 @@ export interface MissionProps {
 }
 export const Mission = (props: MissionProps) => {
     const isSmallScreen = props.screenWidth === ScreenWidths.Sm;
-    const image = (
-        <div className="col lg-col-6 md-col-6 col-12 sm-py2 px2 center">
-            <img src="/images/jobs/map.png" style={{ width: '100%' }} />
-        </div>
-    );
-    const missionStatementStyle = !isSmallScreen ? { height: 364, lineHeight: '364px' } : undefined;
+    const image = <img src="/images/jobs/world-map.svg" style={{ maxWidth: 500, maxHeight: 280 }} />;
```
Comment (fragosti) [inline packages/website/ts/pages/jobs/mission.tsx]: `Image`?

File: packages/website/ts/pages/jobs/open_positions.tsx [hunk 3]
```diff
@@ -45,16 +41,21 @@ export class OpenPositions extends React.Component<OpenPositionsProps, OpenPosit
     }
     public render(): React.ReactNode {
         const isReadyToRender = _.isUndefined(this.state.error) && !_.isUndefined(this.state.jobInfos);
+        const isSmallScreen = this.props.screenWidth === ScreenWidths.Sm;
```
Comment (fragosti) [inline packages/website/ts/pages/jobs/open_positions.tsx]: We also have `utils.isMobileWidth`

File: packages/website/ts/pages/jobs/benefits.tsx (hunk 1 above)
Comment (BMillman19) [reply packages/website/ts/pages/jobs/benefits.tsx]: yea lets just leave it

**Pull request merged**